packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -199,12 +212,12 @@ export class SummarizerNode implements IRootSummarizerNode {
 				0x5df /* Summarize should not be called when not tracking the summary */,
 			);
 			incrementalSummaryContext =
-				this._latestSummary !== undefined
+				this._lastSummaryReferenceSequenceNumber !== undefined
 					? {
 							summarySequenceNumber: this.wipReferenceSequenceNumber,
-							latestSummarySequenceNumber: this._latestSummary.referenceSequenceNumber,
-							// TODO: remove summaryPath
-							summaryPath: this._latestSummary.fullPath.path,
+							latestSummarySequenceNumber: this._lastSummaryReferenceSequenceNumber,
+							// TODO: remove summaryPath.
+							summaryPath: this._summaryHandleId.toString(),
```

Comment:
```suggestion
							summaryPath: this.summaryHandleId,
```

Reply:
@agarwal-navin  --- the type of the summaryPath is a string in `IExperimentalIncrementalSummaryContext`.